src/main/python/keyboard_to_nt.py

```
@@ -1,33 +1,43 @@
 
-    # Wait for connection
     print("Waiting for connection to NetworkTables server...")
     while not ntcoreinst.isConnected():
         time.sleep(0.1)
 
     print("Connected!")
     table = ntcoreinst.getTable("SmartDashboard/keyboard")
 
-    keyboard.hook(lambda e: on_action(e))
-    keyboard.wait()
+    def on_press(key):
```

Ruff (0.14.10)

Missing return type annotation for private function `on_press`

Add return type annotation: `None`

(ANN202)